src/components/LanguageSwitcher.tsx

```diff
@@ -0,0 +1,17 @@
+import Icon from "@/components/Icon";
+import Button from "@/components/Button";
+
+export default function LanguageSwitcher() {
+  return (
+    <Button
+      className="bg-white z-10 w-[78px] h-[44px]"
+      aria-label="change language button"
```

Comment:
Be very careful when using `aria-label` as screen readers will only read that label and skip the rest of the content. This means a blind user or a web crawler will not see what language they are currently in.

It might be better to use `title` here to apply a tooltip for extra information like “เปลี่ยนภาษา”. If your message is descriptive enough to replace the current contents like “เปลี่ยนเป็นภาษาไทย”, apply that to both `title` and `aria-label`.